### PR TITLE
pvaithee07-md updating the caption

### PR DESCRIPTION
**Pull request by pvaithee07**
Updated the caption in pvaithee07-md

Resolves #2

### CODE REVIEW

File: _slides/10-pvaithee07.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello Hello!!!
```
Comment (ericalloyd) [inline _slides/10-pvaithee07.md:6]: LGTM! 👍🏾 Good job!

Comment (ericalloyd) [reply _slides/10-pvaithee07.md:6]:
```suggestion
Hello!!!
```

**Review by ericalloyd — CHANGES_REQUESTED** [review]
We should not duplicate "Hello". Take a look.

File: _slides/10-pvaithee07.md (hunk 1 above)
Comment (pvaithee07) [reply _slides/10-pvaithee07.md:6]: Thanks, I will modify.

**Review by ericalloyd — APPROVED** [review]
👍🏾

**Pull request merged**